js/exporter/jspdf/v3/export_data_grid_3.js

```diff
@@ -24,7 +24,8 @@ function exportDataGrid(doc, dataGrid, options) {
     return new Promise((resolve) => {
         dataProvider.ready().done(() => {
             // TODO: pass rowOptions: { headerStyles: { backgroundColor }, groupStyles: {...}, totalStyles: {...} }
-            const rowsInfo = generateRowsInfo(dataProvider, dataGrid);
+            const rowOptions = options.rowOptions || {};
```

Comment:
'const rowOptions' - excessive variable?
Can we remove it?

Reply:
Ok, I rolled back this excessive variable